exercises/complex-numbers/src/complex_numbers.c

```diff
@@ -0,0 +1,46 @@
+#include "complex_numbers.h"
+
+complex_t c_add(complex_t a, complex_t b)
```

Comment:
I think the parameters should be `const` here as well as in the header file, right?

Reply:
Since these aren't passed by reference, it doesn't really matter, right? It only really matters in the C file (and only because it constrains your implementation).

Reply:
Oh, I see now that the included header includes the `const`s. My preference would be non-`const` everywhere so that we don't lead anyone to believe that the `const` makes a difference, but as long as we're consistent I could be happy either way.

Reply:
Ah, of course; yes I'd agree, although either way is good my preference is to remove `const`, but whichever is chosen, all definitions should be consistent.